src/payload.rs

```diff
@@ -22,18 +23,12 @@ pub fn assert_signed(signature: &str, payload: &[u8]) -> Result<(), SignedPayloa
         }
     };
 
-    let key = PKey::hmac(
+    let mut mac = Hmac::<Sha1>::new_from_slice(
```

Comment:
While we're modifying this, could you please switch to Sha256, and examine the `X-Hub-Signature-256` header rather than `X-Hub-Signature`? The old header is [deprecated](https://docs.github.com/en/webhooks/using-webhooks/validating-webhook-deliveries#troubleshooting). But if you don't want to do it, it's fine, I can do it in a separate PR.